tools/pull_request_hooks/autoLabel.js

```diff
@@ -48,7 +48,7 @@ function check_body_for_labels(body) {
   const labels_to_add = [];
 
   // detect "fixes #1234" or "resolves #1234" in body
-  const fix_regex = /\b(?:fix(?:es|ed)?|resolve[sd]?)\s*#\d+\b/gim;
+  const fix_regex = /\b(?:fix(?:es|ed)?|resolve[sd]?)\s*(?:#\d+|https:\/\/github\.com\/\S+\/issues\/\d+)/gim;
```

Comment:
What if I link an issue in some random repository on github. Will it close that issue as well?

Reply:
closing the issue is something github does, this just changes what labels you get